BaseTools/Source/C/Makefiles/ms.common

```diff
@@ -51,6 +51,25 @@ BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win64
 LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
 SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
 SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
+CFLAGS = $(CFLAGS) /wd4267 /wd4244 /wd4334
+
+!ELSEIF "$(HOST_ARCH)"=="ARM"
+ARCH_INCLUDE = $(EDK2_PATH)\MdePkg\Include\Arm
+BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win32
+LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win32
+SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win32
+SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win32
+
+!ELSEIF "$(HOST_ARCH)"=="AARCH64"
+ARCH_INCLUDE = $(EDK2_PATH)\MdePkg\Include\AArch64
+BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win64
+LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
+SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
+SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
+# Note: These are bit-width conversion related warning suppressions we carry to avoid much more

```

Comment:
I think using "we" here, which I think is a reference to project mu, is confusing.
If you could stop the sentence after "suppressions".

Reply:
Understood, will update.